api/StaticRange.json

```diff
@@ -14,13 +14,14 @@
             "version_added": true
           },
           "firefox": {
-            "version_added": null
+            "version_added": "69",
+            "notes": "In Firefox, <code>StaticRange</code> is only available to privileged code such as the browser itself"
```

Comment:
Not really sure how to interpret this (and I'm not finding any details in the ticket, though I haven't looked too hard).  Can you explain this a little more?

Reply:
Submitting an updated string shortly.

Reply:
Thanks for updating the notes!  Launching Firefox Dev. Ed., it appears that `StaticRange` may actually be exposed already (I launched a console in a new tab and simply typed `StaticRange`).  I have no idea how to call the constructor or use the API in general, but I'd assume being able to call it as such should indicate websites can use it?

Reply:
So, there is a [comment in the IDL](https://dxr.mozilla.org/mozilla-central/source/dom/webidl/StaticRange.webidl?q=staticrange.webidl&redirect_type=direct#13) that says that you can't create a `StaticRange` from web apps. In addition, Firefox doesn't yet implement the constructor. The type is there, but there's no way to make one. I suppose technically the wording should say it that way instead of "only available to privileged code." Personally I don't think the difference is worth worrying about right now but am willing to make the change if someone argues for it.

Reply:
Ah, thanks for linking to that comment!  Well, that sums it up — thanks for clearing up my questions on this.  😄